src/datasets/datasets.controller.ts

```diff
@@ -61,7 +61,7 @@ import { DatablocksService } from "src/datablocks/datablocks.service";
 import { Datablock } from "src/datablocks/schemas/datablock.schema";
 import { CreateDatablockDto } from "src/datablocks/dto/create-datablock.dto";
 import { PartialUpdateDatablockDto } from "src/datablocks/dto/update-datablock.dto";
-import { UpdateQuery } from "mongoose";
+import { Document, UpdateQuery } from "mongoose";
```

Comment:
Lint error - 'Document' is defined but never used

Reply:
Fixed